README.developers.md

```diff
@@ -92,7 +92,7 @@ The overall approach is similar, but we call out the differences below.
 
 ## Commit Messages
 
-Commit messages are an important part of understanding the code base and it's history.
+Commit messages are an important part of understanding the code base and its history.
```

Comment:
@umariqbal-rs Please do not push any changes which is out of the scope of a PR. It will make code review complicated. Each PR should stay focused on a single objective. This is the first time. I am cool with it. But please do **not** this next time.